Sources/SPMBuildCore/PluginInvocation.swift

```diff
@@ -859,13 +859,24 @@ extension ObservabilityMetadata {
 public struct FileLocation: Equatable, CustomStringConvertible {
     public let file: AbsolutePath
     public let line: Int?
+    public let column: Int?
```

Comment:
Eventually I think we will want line/column ranges here (as the .dia format supports) for that's for a later PR.  Just a note about the direction I think we'll end up going here.